overlord/snapstate/handlers.go

```diff
@@ -322,7 +322,7 @@ func (m *SnapManager) installPrereqs(t *state.Task, base string, prereq []string
 			tsSnapd, err = m.installOneBaseOrRequired(st, "snapd", noTypeBaseCheck, defaultSnapdSnapsChannel(), onInFlightErr, userID)
 		})
 		if err != nil {
-			return err
+			return prereqError("system snap", "snapd", err) // XXX
```

Comment:
Why XXX? Anything missing?

Reply:
no, it's a left over, I'll remove it, sorry